modules/local/eukulele/main.nf

```diff
@@ -25,7 +25,7 @@ process EUKULELE {
     """
     rc=0
     mkdir contigs
-    mv ${contigs_fasta} contigs
+    cp ${contigs_fasta} ./contigs/
```

Comment:
Why copy and not move? And if copy, why not symlinking?

Reply:
move will remove the contig from the work directory and it will not be available for other modules (e.g. if EUKulele starts before eggnog, the latter will not be able to use the contig file).
I tried with symlink but it is broken for some reason so I decided to use cp and it worked